integration/test_cluster.py

```diff
@@ -0,0 +1,95 @@
+import pytest
+
+import weaviate
+from weaviate.collections.classes.config import (
+    Configure,
+    DataType,
+    Property,
+)
+from weaviate.util import parse_version_string
+
+
+NODE_NAME = "node1"
+NUM_OBJECT = 10
+
+
+@pytest.fixture(scope="module")
+def client():
+    client = weaviate.connect_to_local()
+    client.collections.delete_all()
+    yield client
+    client.collections.delete_all()
```

Comment:
You need to call `client.close()` once done with it so that resources can be cleaned up gracefully, cheers!

Reply:
you cannot delete all other classes here - other tests are running concurrently and you will delete their data while the test is running.

v3 tests are running sequentially, so it is not a problem there.

The flow should be:
- delete collection for specific test (in case there were leftovers)
- create collection
- do test things
- delete collection

Reply:
Thanks for feedback, I was not aware that test run in parallel in v4, refactored accordingly.